### ISSUE TITLE
Missing from_unixtimestamp

### ISSUE THREAD

**Issue by mirosval** [calogica/dbt-expectations#85]
Hello,

I have updated to dbt `0.19.2` and I'm now getting the following error:

```
Running with dbt=0.19.2
Encountered an error:
Compilation Error
  In dispatch: No macro named 'from_unixtimestamp' found
      Searched for: 'dbt_date.redshift__from_unixtimestamp', 'dbt_date.default__from_unixtimestamp'
```

I've found this commit: https://github.com/calogica/dbt-date/commit/bc5a637868ba5e2fb358e88820cbd5d458d4715c that adds this to `dbt-date` however the currently released version of `dbt-expectations` package is including an older version of that package (though it seems to be up to date on `main`). So I get a version conflict like: 

```
Running with dbt=0.19.2
Encountered an error:
Version error for package calogica/dbt_date: Could not find a satisfactory version from options: ['>=0.3.0', '<0.4.0', '>=0.2.0', '<0.3.0']
```

I'm not sure how it worked before (where that function was defined) or what really broke and I realize that dbt-expectations is probably not the right place to complain about it, but it seems you're in a position to fix it?

**Comment by clausherther** [comment]
Super weird, not sure what's going on there but I'll look into it today. Thanks!

**Comment by clausherther** [comment]
I take it you're on Redshift? I think I see what's happening. dbt-date 0.3.1 added support for Postgres and other non-BQ/Snowflake platforms for that macro, but I forgot to bump the version here. Let me do a bugfix release of dbt-expectations real quick to bump the version of dbt-date to `< 0.4.0` which should fix this (once it's on dbt-hub)

**Comment by clausherther** [comment]
Just released `0.3.5` which should fix this. Once it's updated here https://hub.getdbt.com/calogica/dbt_expectations/latest/ you should be able to do a `dbt deps` and get `dbt-expectations 0.3.5` and `dbt-date 0.3.1`.
